modules/calib3d/perf/perf_undistort.cpp

```diff
@@ -27,4 +27,40 @@ PERF_TEST(Undistort, DISABLED_InitInverseRectificationMap)
     SANITY_CHECK_NOTHING();
 }
 
+using PerfIntType = perf::TestBaseWithParam<std::tuple<int>>;
+PERF_TEST_P(PerfIntType, fisheye_undistortPoints,
+                                            (testing::Values(1e2, 1e3, 1e4)))
```

Comment:
Test cases with parameters 100/1000 is unstable (result is not measurable - sample is too small against overhead, each dedicated run gives different result values)

---

> PerfIntType

It not a good part of the test name (non-descriptive).

---

> `perf::TestBaseWithParam<std::tuple<int>>`

Should be `perf::TestBaseWithParam<int>`

---

Changed in #22028

Reply:
> It not a good part of the test name (non-descriptive).

I was guided by this documentation https://github.com/opencv/opencv/wiki/HowToWritePerfTests

> Test cases with parameters 100/1000 is unstable (result is not measurable - sample is too small against overhead,

What is a minimal allowable process time? It was pretty stable on my machine.
It was important to me to run with different amout of points to show how computation time depends on it.